packages/dashboard/src/utils/constants.ts

```diff
@@ -3,32 +3,43 @@ import chainIDtoNameJson from "src/assets/chainIDtoName.json";
 export const EMOTION_KEY = "trfl";
 export const COLOR_SCHEME_KEY = "trfl.dash.color-scheme";
 
-export const INTERACTIVE_RPC_METHODS_ARR = [
-  "eth_signTransaction",
+export const decodableRpcMethodsArr = [
```

Comment:
Ooh maybe your Prism method "language" detection thing should use these constants instead of hardcoding the methods as string literals

Reply:
It's [using the type of these constants](https://github.com/trufflesuite/truffle/blob/68d9d5fcc1014a6aa5da99372373479139ec54b7/packages/dashboard/src/components/composed/RPCs/RPC/Details/Expanded/DecodedParams.tsx#L15) (`type DecodableRpcMethod` is just `typeof decodableRpcMethodsArr[number]`). 

I figured `method === "eth_sendTransaction"` is more understandable than `method === decodableRpcMethodsArr[0]`.